django_project/frontend/serializers/metrics.py

```diff
@@ -299,3 +299,39 @@ def get_age_group(self, obj) -> dict:
         )
 
         return age_groups_totals
+
+
+class TotalAreaVSAvailableAreaSerializer(serializers.ModelSerializer):
+    """
+    Serializer class for serializing total area and available area.
+    """
+    get_area = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Taxon
+        fields = ["get_area", "common_name_varbatim"]
+
+    def get_get_area(self, obj) -> dict:
```

Comment:
```suggestion
    def get_get_area(self, obj) -> list:
```

It should be a list, because you are returning list of queryset.